### PR TITLE
Organ poking, biocharger (cybernetic heart sidegrade)

### PR DESCRIPTION
**Pull request by MadmanMartian**
The biochargerl allows you to use a human body to charge a cell.

May be used for something in the future. Perhaps cybernetics.

### CODE REVIEW

File: code/modules/organs/organ_objects.dm [hunk 1]
```diff
@@ -137,6 +137,34 @@
 	dead_icon = "heart-off"
 	organ_type = /datum/organ/internal/heart
 
+/obj/item/organ/internal/heart/cell
+	name = "biocharger"
+	icon_state = "heart=cell"
```
Comment (DamianX) [inline code/modules/organs/organ_objects.dm]: `heart=cell` => `heart-cell`
100% tested?

Comment (MadmanMartian) [reply code/modules/organs/organ_objects.dm]: oof

File: code/modules/organs/internal/heart/heart.dm [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	min_bruised_damage = 15
+	min_broken_damage = 30
+	var/obj/item/weapon/cell/cell
+
+/datum/organ/internal/heart/cell/New()
+	..()
+	if(!cell)
+		cell = new /obj/item/weapon/cell/empty()
+
+/datum/organ/internal/heart/cell/Life()
```
Comment (MadmanMartian) [inline code/modules/organs/internal/heart/heart.dm:21]: Turns out the generic organ Life() call is never called.

Adding now.

File: code/modules/organs/internal/heart/heart.dm [hunk 3]
```diff
@@ -0,0 +1,25 @@
+//heart upgrades
+
+/datum/organ/internal/heart //This is not set to vital because death immediately occurs in blood.dm if it is removed.
```
Comment (jknpj) [inline code/modules/organs/internal/heart/heart.dm:3]: needs its own get_cell() maybe

Comment (jknpj) [reply code/modules/organs/internal/heart/heart.dm:3]: i'm retarde

File: vgstation13.dme [hunk 4]
```diff
@@ -71,6 +71,7 @@
 #include "code\hub.dm"
 #include "code\names.dm"
 #include "code\world.dm"
+#include "code\modules\organs\internal\heart\heart.dm"
```
Comment (jknpj) [inline vgstation13.dme:74]: Intigracy is going to FREAK

Comment (MadmanMartian) [reply vgstation13.dme:74]: What the kfuck

**Comment by MadmanMartian** [comment]
why did you merge this with that bad sort?

**Pull request merged**